### ISSUE TITLE
Long node names truncated

### ISSUE THREAD

**Issue by m-emm** [VisualDataWeb/WebVOWL#82]
In ontologies with a bit longer node names (longer that about 10 characters), the node names are truncated, and often the resulting graph is barely useable because of that.
This can be easily seen when looking at the FOAF ontology on the nodes "Online Account", "Online Chat Account", "Online E-commerce Account".

One possible solution might be to break camel case or spaced labels into multiple lines.

**Comment by steffen-l** [comment]
Good point! We are aware of it, and it is already on our issue list. We will likely implement a similar solution as in LD-VOWL also for WebVOWL in the future: http://vowl.visualdataweb.org/ldvowl/ So this will likely be included in some future release. For now, you have the tooltips with the full name when you hover the element.

**Comment by handrews** [comment]
@steffen-l would you be receptive to pull requests?  I didn't see a contributors' guide but perhaps I overlooked it somewhere.  Not promising anything but I am really liking this tool and would like to help it along if I get a chance.

**Comment by steffen-l** [comment]
Yes, we are generally fine with pull requests and will likely accept them (if we don't have any bigger objections). Just go ahead! Contributions are appreciated.

However, note that for this issue here ("long node names truncated"), we already implemented a solution in part that will soon be completed and pushed - likely before end of May).

**Comment by handrews** [comment]
@steffen-l great to hear on both counts.  I'll hold off on this one and wait for the next release.

**Comment by imaneelarabi** [comment]
Is there any news on this?

**Comment by nandikajain** [comment]
@steffen-l @vitalis-wiens 
The issues arises here in [src/webvowl/js/util/](https://github.com/VisualDataWeb/WebVOWL/tree/master/src/webvowl/js/util):
```
CenteringTextElement.prototype.addTextline = function ( text, style, prefix, postfix ){
  var truncatedText = textTools.truncate(text, this._textBlock().datum().textWidth(), style);
```
```
AbsoluteTextElement.prototype.addTextline = function ( text, style, yShift, prefix, postfix ){
  var truncatedText = textTools.truncate(text, this._textBlock().datum().textWidth(yShift), style);
```
Here, the text is truncated. If we split the text based on the white spaces and then add line breaks, that could possibly solve this issue.
